tree/30_generic_methods/command_execution.cf

```diff
@@ -21,27 +21,29 @@
 # @action
 #
 # @parameter command_name  Command name
-# 
+#
 # @class_prefix command_execution
 # @class_parameter command_name
 
 bundle agent command_execution(command_name)
 {
   vars:
-      "old_class_prefix"  string => canonify("command_execution_${command_name}");
+      "old_class_prefix"  string => canonify("${report_data.canonified_directive_id}_command_execution_${command_name}");
```

Comment:
This is tricky we are changing the expected outcome classes, maybe we should add a third one ?